docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     volumes:
       - "dbdata:/var/lib/postgresql/data"
   es:
-    image: "elasticsearch:1"
+    image: "elasticsearch:6"
```

Comment:
This won't work since there isn't a tag for 6 in the elasticsearch docker hub. Maybe change it to: 
```yaml
image: "elasticsearch:6.8.1"
```